.github/workflows/main.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Install tox
         run: |
           python -m pip install --upgrade pip
-          pip install tox
+          pip install tox==3.28.0
```

Comment:
Why this pin? Leave a comment explaining why, but better yet, just fix the problem for tox 4.0... are you getting any errors?

Reply:
Looking at the file, it seems you need to rename `passenv` to `pass_env`:

https://github.com/ESSS/serialchemy/blob/943a03d505565f8e9421dd4b439ff55c79162381/tox.ini#LL6-L6C8

Reply:
Te fix was actually replace white spaces by comma. I've removed the pin.

Reply:
Argh right, that was the change! hehehe

FYI you could also have used one var per-line.